Lab-4/patrik_svensson_lab_function_approximation.ipynb

```diff
@@ -0,0 +1,286 @@
+            "text/plain": [
+              "<Figure size 640x480 with 1 Axes>"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          }
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "kl5VpJcfMgTf",
+        "colab_type": "text"
+      },
+      "source": [
+        "# **Results**\n",
+        "The results of the implemetation can be seen in the graph in the previous section."
```

Comment:
You have not added any graph related to error and convergence. This needs to be added.